src/opentimelineio/imageSequenceReference.cpp

```diff
@@ -120,13 +120,22 @@ ImageSequenceReference::ImageSequenceReference(std::string const& target_url_bas
     }
 
     bool ImageSequenceReference::read_from(Reader& reader) {
+
+        int64_t start_frame_value = 0;
+        int64_t frame_step_value = 0;
+        int64_t frame_zero_padding_value = 0;
+
         auto result = reader.read("target_url_base", &_target_url_base) &&
             reader.read("name_prefix", &_name_prefix) &&
             reader.read("name_suffix", &_name_suffix) &&
-            reader.read("start_frame", &_start_frame) &&
-            reader.read("frame_step", &_frame_step) &&
+            reader.read("start_frame", &start_frame_value) &&
+            reader.read("frame_step", &frame_step_value) &&
             reader.read("rate", &_rate) &&
-            reader.read("frame_zero_padding", &_frame_zero_padding);
+            reader.read("frame_zero_padding", &frame_zero_padding_value);
+
+        _start_frame = static_cast<int>(start_frame_value);
```

Comment:
I suggest we just change the `start_frame` ivar to be `int64_t`. I think it was a mistake I didn't do that in the first place.